tests/testthat/test-check-cardinalities.R

```diff
@@ -48,24 +40,18 @@ test_that("check_cardinality_...() functions are checking the cardinality correc
 
   # expect specific errors and sometimes specific output due to errors ---------------
 
-  pmap(
-    list(
-      d1_src,
-      d2_src,
-      card_0_n_d1_d2_names
-    ),
-    ~ expect_known_output(
-      expect_dm_error(
-        check_cardinality_0_n(
-          parent_table = ..1,
-          pk_column = a,
-          child_table = ..2,
-          fk_column = a
-        ),
-        class = "not_subset_of"
+  # FIXME: Regarding PR #313: the known output changes, depending on if it's on a DB or locally
+  expect_known_output(
```

Comment:
Can you please switch to `verify_output()` everywhere?

Reply:
Sure, but that doesn't solve the problem at hand: e.g. for SQLite the line `# A tibble: 1 x 2` is missing with comparison to the local `src`.

Reply:
Good point. Can you please introduce `verify_output_src()` ?

Reply:
I just saw that we introduced a workaround at some point for trimming the `tbl_sql` print-output like so:
```r
if (is_this_a_test()) {
  tbl_sql <- dbplyr::tbl_sql
  tbl_sum.tbl_sql <- function(x, ...) c()
  vctrs::s3_register("tibble::tbl_sum", "tbl_sql", tbl_sum.tbl_sql)
}
```
Adding two more lines here solves the issue (as long as we don't have tibbles with more than 10 rows):
```r
if (is_this_a_test()) {
  tbl_sql <- dbplyr::tbl_sql
  tbl_sum.tbl_sql <- function(x, ...) c()
  vctrs::s3_register("tibble::tbl_sum", "tbl_sql", tbl_sum.tbl_sql)
  tbl_sum.tbl_df <- function(x, ...) c()
  vctrs::s3_register("tibble::tbl_sum", "tbl_df", tbl_sum.tbl_df)
}
```
the head of the output then looks the same and we don't need `verify_output_src()`.

Caveat: if we execute a test in the console with `testthis:::test_this()`, afterwards neither `tbl_sql` nor `tbl_df` will look like we are used to (for `tbl_sql` already the case). (maybe we can somehow restore the normal `tbl_sum`-methods after tests are done?)

Is that ok for you?